File: poo-noite/aula04/edu/curso/Carro.java
package edu.curso;

public class Carro {

    private Motor motor = new Motor();

    Pneu p1;
    Pneu p2;
    Pneu p3;
    Pneu p4;
    Pneu p5;

    public Carro() { 
        p1 = new Pneu(15);
        p2 = new Pneu(15);
        p3= new Pneu(15);
        p4 = new Pneu(15);
        p5 = new Pneu(15);
        motor = new Motor();
    }
    
}

File: prog2/aula12/carro.py
class Carro:
    roda
    cor
    motor
    numero_portas
    numero_assentos
    ligado
    velocidade

    def ligar(self):
        print("Carro ligado ...")
        self.ligado = True
        self.motor = True

    def iluminar(self):
        print("Iluminando ...")

    def mover(self):
        if self.ligado == True:
            print("Se movendo ...")
            self.velocidade = 20
        else:
            print("O carro ainda esta desligado")